src/main/resources/application-test.yml

```
@@ -0,0 +1,29 @@
+server :
```

YAMLlint (1.35.1)

[warning] too many spaces before colon

(colons)